File: Código-conta-bancaria.py
class ContaBancaria:
 def __init__(self, titular, saldo_inicial):
        self.titular = titular
        self.saldo = saldo_inicial

 def depositar(self, valor):
        self.saldo += valor
        print(f"Depósito de R$ {valor} realizado com sucesso!")

 def sacar(self, valor):
        if valor <= self.saldo:
            self.saldo -= valor
            print(f"Saque de R$ {valor} realizado com sucesso!")
        else:
            print("Saldo insuficiente!")

 def exibir_saldo(self):
        print("\n--- Informações da Conta ---")
        print(f"Titular: {self.titular}")
        print(f"Saldo atual: R$ {self.saldo}")

File: Código-conta_bancaria.java
package construtor;

public class conta_bancaria {
	private String titular;
	private double saldo;

	public conta_bancaria(String titular, double saldoInicial) {
		this.titular = titular;
		this.saldo = saldoInicial;
	}

	public void depositar(double valor) {
		saldo += valor;
		System.out.println("Depósito de R$ " + valor + " realizado com sucesso!");
	}

	public void sacar(double valor) {
		if (valor <= saldo) {
			saldo -= valor;
			System.out.println("Saque de R$ " + valor + " realizado com sucesso!");
		} else {
			System.out.println("Saldo insuficiente!");
		}
	}

	public void exibirSaldo() {
		System.out.println("\n--- Informações da Conta ---");
		System.out.println("Titular: " + titular);
		System.out.println("Saldo atual: R$ " + saldo);
	}
}
